v2/go.mod

```diff
@@ -0,0 +1,25 @@
+module github.com/go-pkgz/lcw/v2
+
+go 1.21
+
+toolchain go1.21.6
```

Comment:
this better to be removed, we don't want to enforce toolchain version especially for the library